README.rst

```diff
@@ -210,6 +210,11 @@ Then, use either of the following to invoke the test suite::
 
 This uses example data found in the ``cxarchive/tests/data`` directory.
 
+Usage
+-----
+ * [Content API](/docs/content_api_doc.rm)
```

Comment:
The rst syntax you are looking for is `` `Content API <docs/search_api_doc.md>`_ ``